blog/_posts/2017-11-30-bloop-release.md

```diff
@@ -0,0 +1,190 @@
+---
+layout: blog-detail
+post-type: blog
+by: Jorge Vicente Cantero and Martin Duhem
+title: "Meet bloop, a fast tool to compile and test your project"
+---
+
+Martin Duhem and I are happy to announce `bloop`, a tool that Scala developers
+can use to compile and test their projects faster without being tied to their
+stock build tool.
+
+We started this project after the feedback we got from this tweet, in which some of you encouraged
+us to work on compiler/build performance:
+
+<blockquote class="twitter-tweet" data-lang="en-gb"><p lang="en" dir="ltr">
+If the Scala Center had 1 engineer free for 1 month what is the most meaningful thing we could do to help your day-to-day Scala development?
+</p>&mdash; Scala (@scala_lang) <a href="https://twitter.com/scala_lang/status/928242963091808256">8th November 2017</a></blockquote><script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+## What is bloop?
+
+Bloop is a command-line tool for fast edit/compile/test workflows. Its primary
+goal is to compile and test your project faster than your stock build tool,
```

Comment:
This still sounds like you're trying to replace the "stock build tool." 

How about this instead:

"Bloop is a command-line tool for fast edit/compile/test workflows. Its primary goal is to give you the fastest possible experience specifically in the edit/compile/test loop. Bloop is not meant to replace your current build system."

Reply:
maybe also add some wording like ", because bloop is intended for interactive use when you are in a tight edit/build/test loop"? if that is the intent behind the name

Reply:
yeah, I had no idea what the name meant, otherwise